cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -69,10 +69,13 @@ These functions return a single value.
 | `size(input :: STRING?) :: (INTEGER?)` | Returns the number of Unicode characters in a string.
 1.1+| <<functions-startnode,`startNode()`>>  | `startNode(input :: RELATIONSHIP?) :: (NODE?)` | Returns the start node of a relationship.
 1.1+| <<functions-toboolean,`toBoolean()`>>  | `toBoolean(input :: STRING?) :: (BOOLEAN?)` | Converts a string value to a boolean value.
+1.1+| <<functions-tobooleanornull,`toBooleanOrNull()`>>  | `toBooleanOrNull(input :: ANY?) :: (BOOLEAN?)` | Converts a value to a boolean value, or null if the value cannot be converted.
 1.2+| <<functions-tofloat,`toFloat()`>>  | `toFloat(input :: NUMBER?) :: (FLOAT?)` | Converts an integer value to a floating point value.
```

Comment:
Technically the `NUMBER` could be either an integer or a float. I would suggest rephrasing it to `Converts a number value to a floating point value.` to match the corresponding line for `toInteger`